third_party/rust/chromium_crates_io/gnrt_config.toml

```diff
@@ -130,6 +141,13 @@ if (target_cpu == "x64" || target_cpu == "x86") {
 }
 ''' }
 
+[crate.cpufeatures]
```

Comment:
Would be nice to keep the crates configs in alphabetical order

Reply:
Fixed in fb45c252f3429fe09d4ba7ed3a41867b3c585fde